WatsonDeveloperCloud/Dialog/Dialog.swift

```diff
@@ -19,49 +19,59 @@ import Alamofire
 import AlamofireObjectMapper
 import ObjectMapper
 
-/// The IBM Watson™ Dialog service provides a comprehensive, robust, platform for managing conversations between virtual agents and users through an application programming interface (API). These conversations are commonly referred to as dialogs.
+/**
+ The IBM Watson™ Dialog service provides a comprehensive, robust, platform for managing
+ conversations between virtual agents and users through an application programming
+ interface (API). These conversations are commonly referred to as dialogs.
+ */
 public class Dialog: WatsonService {
-    
+
     // The shared WatsonGateway singleton.
     let gateway = WatsonGateway.sharedInstance
-    
+
     // The authentication strategy to obtain authorization tokens.
-    var authStrategy: AuthenticationStrategy
-    
+    let authStrategy: AuthenticationStrategy
+
+    // TODO: comment this initializer
     public required init(authStrategy: AuthenticationStrategy) {
         self.authStrategy = authStrategy
     }
-    
+
+    // TODO: comment this initializer
     public convenience required init(username: String, password: String) {
         let authStrategy = BasicAuthenticationStrategy(tokenURL: Constants.tokenURL,
             serviceURL: Constants.serviceURL, username: username, password: password)
         self.init(authStrategy: authStrategy)
     }
+
```

Comment:
Ah, okay. Now I see. I don't think I've stumbled across any other code in the SDK that requires a date formatter, but I will keep my eyes open as I continue to make improvements to the different services.